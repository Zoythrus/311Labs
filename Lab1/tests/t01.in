counting
8 45
13
19
12
3
17
23
45
24
