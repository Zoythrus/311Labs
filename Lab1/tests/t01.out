3
12
13
17
19
23
24
45
